### PR TITLE
Add Bluetooth Ledger state

### PR DESCRIPTION
**Pull request by lubej**

### CODE REVIEW

File: src/app/lib/ledger.ts [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import { ContextSigner } from '@oasisprotocol/client/dist/signature'
```
Comment (lukaw3d) [inline src/app/lib/ledger.ts:1]: I think commits could be redone much better, e.g.:
- Add capacitor dependencies for mobile app support (package, yarn.lock)
- Differentiate USB and Bluetooth Ledger types (affects some existing tests)
- Add state for enumerating Bluetooth Ledger devices
- Add Bluetooth Ledger tests
- Add changelog fragment

File: src/app/state/transaction/saga.test.ts [hunk 2]
```diff
@@ -7,16 +7,22 @@ import { DeepPartialRootState } from 'types/RootState'
...
 import { Wallet, WalletType } from '../wallet/types'
-import { doTransaction, submitParaTimeTransaction, getAllowanceDifference, setAllowance } from './saga'
+import { doTransaction, getAllowanceDifference, setAllowance, submitParaTimeTransaction } from './saga'
+import { addressToPublicKey } from '../../lib/helpers'
+import BleTransport from '@oasisprotocol/ionic-ledger-hw-transport-ble/lib'
 
-const makeState = (wallet: Partial<Wallet>): DeepPartialRootState => {
+const makeState = (
+  wallet: Partial<Wallet>,
+  rootState: Partial<DeepPartialRootState> = {},
```
Comment (lukaw3d) [inline src/app/state/transaction/saga.test.ts:18]: is DeepPartialRootState not correctly partial? :O

Comment (lubej) [reply src/app/state/transaction/saga.test.ts:18]: Nice catch, will remove.

File: src/app/lib/__tests__/ledger.test.ts [hunk 3]
```diff
@@ -31,6 +37,30 @@ describe('Ledger Library', () => {
     jest.resetAllMocks()
   })
 
+  describe('BLE Ledger', () => {
+    it('should support Bluetooth', async () => {
+      ;(BleTransport.isEnabled as jest.Mock).mockResolvedValue(true)
```
Comment (lukaw3d) [inline src/app/lib/__tests__/ledger.test.ts:42]:
```suggestion
      jest.mocked(BleTransport.isEnabled).mockResolvedValue(true)
```

**Review by lukaw3d — APPROVED** [review]

**Pull request merged**